models/advoc/train_evaluate.py

```diff
@@ -212,22 +214,22 @@ def infer(fps, args):
     x_magspec, x_wav = decode_extract_and_batch(
       fps,
       batch_size=args.infer_batch_size,
-      subseq_len= model.subseq_len,
+      slice_len= model.subseq_len,
       audio_fs=model.audio_fs,
       audio_mono=True,
-      audio_normalize=args.audio_normalize,
+      audio_normalize=args.data_normalize,
       decode_fastwav=args.data_fastwav,
       decode_parallel_calls=4,
       extract_type='magspec',
       extract_parallel_calls=8,
       repeat=False,
       shuffle=False,
       shuffle_buffer_size=None,
-      subseq_randomize_offset=False,
-      subseq_overlap_ratio=0.,
-      subseq_pad_end=True,
+      slice_randomize_offset=False,
+      slice_overlap_ratio=0.,
+      slice_pad_end=True,
```

Comment:
this one should probably be equal to `args.data_slice_pad_end` because this determines if the model has ever seen padding